packages/server-core/src/media/static-resource/static-resource.hooks.ts

```diff
@@ -76,27 +76,25 @@ export default {
 
   before: {
     all: [
+      iff(isProvider('external'), verifyScope('static_resource', 'read')),
```

Comment:
Should this be moved to find and get? As-is, this would require read and write scopes to do a write, and I don't know if that's what we want.